core/src/main/java/io/questdb/ServerConfiguration.java

```diff
@@ -67,4 +67,9 @@ default String getReleaseType() {
 
     default void init(CairoEngine engine, FreeOnExit freeOnExit) {
     }
+
+    default boolean isLineTcpEnabled() {
```

Comment:
why do we need this?

Reply:
no idea... I think it was an artifact from a merge